rackcity/views/power_views.py

```diff
@@ -443,15 +373,22 @@ def toggle_pdu_power(asset, asset_port_number, goal_state):
 @api_view(["POST"])
 @permission_classes([IsAuthenticated])
 def chassis_power_status(request):
-    data = JSONParser().parse(request)
     try:
-        chassis_hostname, blade_slot = get_chassis_power_request_parameters(data)
-    except ChassisPowerManagementException as error:
+        chassis, blade_slot = get_chassis_power_request_parameters(request)
+    except PowerManagementException as error:
         return JsonResponse(
             {"failure_message": Status.ERROR.value + str(error)},
             status=HTTPStatus.BAD_REQUEST,
         )
-    result, exit_status = make_bcman_request(chassis_hostname, str(blade_slot), "")
+    except UserPowerPermissionException as error:
+        return JsonResponse(
+            {
+                "failure_message": Status.AUTH_ERROR + AuthFailure.POWER.value,
```

Comment:
I was wrong last time but I THINK this should be `Status.AUTH_ERROR.value`

Reply:
UR SO RIGHT